src/index.html

```diff
@@ -8,6 +8,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="wrapper">
+      <header class="header"></header>
+      <main class="main">
+        <div class="main__block block">
+          <div class="block__stars stars stars--0">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="block__stars stars stars--1">
+            <div class="stars__star stars__star-is-active"></div>
```

Comment:
Remove `stars__star-is-active` everywhere, use `nth:child`

```suggestion
            <div class="stars__star"></div>
```